### PR TITLE
[url_launcher]: Fix a grammar error in the comments of launchUrl method

### PR DESCRIPTION
**Pull request by dshukertjr**
I found a grammer error in the comments of `launchUrl()` method. This PR fixes the error.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by dshukertjr** [comment]
The [CLA](https://cla.developers.google.com/) link isn't opening for me. Will check back in a few hours to see if it starts working again.

File: packages/url_launcher/url_launcher/CHANGELOG.md [hunk 1]
```diff
@@ -1,455 +1,456 @@
 ## NEXT
 
-* Updates support matrix in README to indicate that iOS 11 is no longer supported.
-* Clients on versions of Flutter that still support iOS 11 can continue to use this
+- Updates support matrix in README to indicate that iOS 11 is no longer supported.
```
Comment (stuartmorgan) [inline packages/url_launcher/url_launcher/CHANGELOG.md:3]: Please don't edit the changlog with tooling that is configured in a way that will rewrite the style of the entire file.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks for the contribution!

This will need a version change:
> - [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy](https://dart.dev/tools/pub/versioning), or this PR is [exempt from version changes](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates).

File: packages/url_launcher/url_launcher/CHANGELOG.md (hunk 1 above)
Comment (dshukertjr) [reply packages/url_launcher/url_launcher/CHANGELOG.md:3]: Oops, sorry for that. Will fix it right now.

**Review by stuartmorgan — APPROVED** [review]
LGTM, thanks!

**Comment by stuartmorgan** [comment]
I missed that this is missing an actual version change in the pubspec.yaml (or it was there, and was lost in a merge); it will need that in order to land.

**Comment by dshukertjr** [comment]
Oops, of course. Sorry for missing that. Will do it right now

**Comment by dshukertjr** [comment]
Done!

**Review by bparrishMines — APPROVED** [review]
LGTM

**Pull request merged**